core/tests/ui/e2e/account/wishlist-details.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { expect, test } from '~/tests/fixtures';
+import { getTranslations } from '~/tests/lib/i18n';
+import { TAGS } from '~/tests/tags';
+
+test.describe('Wishlist Details', () => {
```

Comment:
🍹 We already get context in the test runs which file we are running tests against so it's a little redundant to wrap tests in a `.describe`. I tend to lean on `.describe` blocks to segment testing edge-cases or different branches of logic like you do with the actions below.

Reply:
@chanceaclark Yeah, I'm not super opinionated about this - the main reason I've been doing it this way is it makes it easier to run the whole group of tests using the playwright VSCode extension if we wrap the whole suite in a `test.describe`, just a small quality of life improvement. Otherwise, you have to manually browse to the test explorer, find the file, and then run it that way instead of running it directly from the code.

Reply:
This would be particularly useful if we ended up needing domain tags to allow BC services to run specific Catalyst e2e tests during their CI, instead of annotating every test in a suite, we just annotate the root `test.describe`